ps_categorytree.php

```diff
@@ -238,6 +238,11 @@ public function renderForm()
                                 'value' => static::CATEGORY_ROOT_CURRENT_PARENT,
                                 'label' => $this->getTranslator()->trans('Current category, unless it has no subcategories, in which case the parent category of the current category is used', [], 'Modules.Categorytree.Admin'),
                             ],
+			    [
+                                'id' => 'category_depth_1',
+                                'value' => 4,
```

Comment:
This magic value requires its own `static::CATEGORY_ROOT_FOO` (see how other options are handled)